From aa63712b18b79b983e3d9cd7ce5105d0ec7ac262 Mon Sep 17 00:00:00 2001
Message-ID: <aa63712b18b79b983e3d9cd7ce5105d0ec7ac262.1754900600.git.sam@gentoo.org>
From: Andreas Stieger <Andreas.Stieger@gmx.de>
Date: Fri, 18 Jul 2025 09:39:50 +0200
Subject: [PATCH 1/3] Fix implicit declaration of gpa_keytable_ensure

* src/cm-piv.c: Fix implicit declaration of gpa_keytable_ensure.
--- a/src/cm-piv.c
+++ b/src/cm-piv.c
@@ -41,6 +41,7 @@
 #include "gtktools.h"
 #include "convert.h"
 #include "gpa-key-details.h"
+#include "keytable.h"
 
 #include "cm-object.h"
 #include "cm-piv.h"
-- 
2.50.1

